lib/util/cache/repository/types.ts

```diff
@@ -28,13 +28,16 @@ export interface BranchUpgradeCache {
 
 export interface BranchCache {
   automerge: boolean;
+  branchFingerprint?: string;
   branchName: string;
   isModified: boolean;
+  /**
+   * Parent commit of branch sha (latest branch commit)
+   */
+  parentSha: string | null;
   prNo: number | null;
   sha: string | null;
```

Comment:
can you add a tsdoc comment to the other fields too, so it's clear what they are for?